.github/workflows/sql.yml

```diff
@@ -0,0 +1,65 @@
+name: SQL
+
+on:
+  push:
+    branches:
+      - main
+  pull_request: {}
+
+jobs:
+  mysql:
+    name: ${{ matrix.database.name }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        database:
+          - {name: MySQL 5.7,      image: "mysql:5.7"}
+          - {name: MySQL 8.0,      image: "mysql:8.0"}
```

Comment:
Are these two really all MySQL versions we support?

Reply:
There is also `mysql:latest`, so which version do you want to add?